pkg/numaresourcesscheduler/manifests/yaml/deployment.yaml

```diff
@@ -42,12 +42,9 @@ spec:
               drop: ["ALL"]
           image: ${IMAGE}
           resources:
-            limits:
-              cpu: "600m"
-              memory: "1200Mi"
             requests:
-              cpu: "600m"
-              memory: "1200Mi"
+              cpu: "150m"
+              memory: "500Mi"
```

Comment:
I think we need to check how much normally the scheduler would actually consume these resources before setting a low threshold as I learned it may affect the performance if it is always lower than the actual consumption. For that we probably need to test that on several machines just to get an idea of the expected resources on medium scaled envs